### PR TITLE
sub2unlock

### PR DESCRIPTION
**Pull request by elidevz**
<details> <summary>NOTICE</summary>
I dedicate any and all copyright interest in this software to the
public domain. I make this dedication for the benefit of the public at
large and to the detriment of my heirs and successors. I intend this
dedication to be an overt act of relinquishment in perpetuity of all
present and future rights to this software under copyright law.
</details>

Fixes: Link to issue

- [x] I made sure there are no unnecessary changes in the code*
- [x] Tested on Chromium- Browser OS
- [x] Tested on Firefox

\* indicates required

### CODE REVIEW

File: src/js/injection_script.js [hunk 1]
```diff
@@ -1996,6 +1996,21 @@ ensureDomLoaded(()=>{
 	domainBypass("work.ink", () => {
 		ifElement("#redirect-button", () => openFinalLink())
 	})
+	
+	domainBypass("sub2unlock.com", () => {
+	    const url = document.URL;
+	    if (url.includes("sub2unlock.com/link/unlock")) {
```
Comment (reashetyrr) [inline src/js/injection_script.js:2002]: Might as well check for "link/unlock"

File: src/js/injection_script.js [hunk 2]
```diff
@@ -1996,6 +1996,21 @@ ensureDomLoaded(()=>{
...
+		console.log('URL is unlocked, skipping...')
+		return;
+	    } else {
+		console.log('URL is not unlocked, continuing...')
+		const urlSplit = url.split("/");
+		const urlLast = urlSplit[urlSplit.length - 1];
```
Comment (reashetyrr) [inline src/js/injection_script.js:2008]: You can use .pop() to get the last item back

**Comment by elidevz** [comment]
seems fine + firefox test worked

**Comment by lostdusty** [comment]
Could you give an example link?

**Comment by elidevz** [comment]
https://sub2unlock.com/0yYfV

**Pull request merged**